client/my-sites/stats/stats-comments/index.jsx

```diff
@@ -123,8 +123,8 @@ class StatsComments extends Component {
 		const commentsPosts = get( commentsStatsData, 'posts' );
 		const noData = ! commentsAuthors;
 		const selectOptions = [
-			{ value: 'top-authors', label: translate( 'Comments By Authors' ) },
-			{ value: 'top-content', label: translate( 'Comments By Posts & Pages' ) },
+			{ value: 'top-authors', label: translate( 'Comments By authors' ) },
+			{ value: 'top-content', label: translate( 'Comments By posts & pages' ) },
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.